libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -167,6 +171,13 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
     const changedProjectsOnly: boolean = this._isIncrementalBuildAllowed && this._changedProjectsOnly.value;
 
     const terminal: Terminal = new Terminal(this.rushSession.terminalProvider);
+    if (showTimeline) {
+      const { ConsoleTimelinePlugin } = await import('../../logic/operations/ConsoleTimelinePlugin');
+      new ConsoleTimelinePlugin(terminal).apply(this.hooks);
+    }
+    // Do the standard summary
+    new OperationResultSummarizerPlugin(terminal).apply(this.hooks);
```

Comment:
This should move to before the action hooks in case a plugin wants to schedule its taps relative to these plugins.